src/GraphQL/Execution/IExecutionStrategySelector.cs

```diff
@@ -0,0 +1,11 @@
+namespace GraphQL.Execution
+{
+    /// <summary>
+    /// Returns an instance of an <see cref="IExecutionStrategy"/> for a specified operation type.
```

Comment:
```suggestion
    /// Returns an instance of an <see cref="IExecutionStrategy"/> for a specified <see cref="ExecutionContext"/>.
```

Reply:
The same question here - "a" or "the" before "specified"?

Reply:
I don't know for sure.  Either way I think would be proper English.